### PR TITLE
add some more material (preliminary provided in "tuto2" directory)

### PR DESCRIPTION
**Pull request by ghost**
This pull-request offers some new material
(preliminarily, in the form of a new tutorial).

Feel free to reuse that information as you see fit.

(e.g. drop redundant information (the one that is already being covered elsewhere))

(e.g. make the style of the new material consistent with the original content)

(...)

### CODE REVIEW

File: tuto2/_CoqProject [hunk 1]
```diff
@@ -0,0 +1,6 @@
+-R theories/ Tuto
+-I src
+
+theories/Test.v
+src/demo.ml4
+src/tuto_plugin.mlpack
```
Comment (ybertot) [inline tuto2/_CoqProject]: Can you add a new line here.

File: tuto2/src/demo.ml4 [hunk 2]
```diff
@@ -0,0 +1,356 @@
+(* This will be compilable once the following pull-requests:
+   - https://github.com/coq/coq/pull/759
+   - https://github.com/coq/coq/pull/761
+   will be merged.
+
```
Comment (ybertot) [inline tuto2/src/demo.ml4]: Can you remove the previous four lines: the pull request were merged AFAICT

File: tuto2/_CoqProject (hunk 1 above)
Comment (ghost) [reply tuto2/_CoqProject]: Ok. See commit a7bf179.

File: tuto2/src/demo.ml4 (hunk 2 above)
Comment (ghost) [reply tuto2/src/demo.ml4]: Ok. See commit a7bf179.

**Comment by ghost** [comment]
Ok. I've dropped the whole comment at the top of `demo.ml4` as it makes little sense to keep it public.

Anything else need to be done?

**Comment by ybertot** [comment]
I am in a meeting now, I will get back to you later (around 17:00). 

Yves

**Review by ybertot — APPROVED** [review]

**Pull request merged**